app/controllers/certificates_controller.rb

```diff
@@ -12,7 +12,6 @@ def create
     if @upload.valid?
       redirect_to certificates_path
     else
-      # FIXME add error messages to view
       Rails.logger.info(@upload.errors.full_messages)
```

Comment:
We can get rid of the error message now. Doesn't have to be done in this PR though.